### PR TITLE
[Browser rendering] Add notes emphasing --remote is required for local development

### PR DESCRIPTION
**Pull request by meddulla**

### CODE REVIEW

File: content/browser-rendering/get-started/browser-rendering-with-DO.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ export class Browser {
 
 ## 7. Test
 
-Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker locally before deploying to Cloudflare's global network.
+Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker locally before deploying to Cloudflare's global network. Take into account there's no `local` mode support so `--remote` is required.
```
Comment (celso) [inline content/browser-rendering/get-started/browser-rendering-with-DO.md]: that into account **that** there's

File: content/browser-rendering/get-started/browser-rendering-with-DO.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ export class Browser {
 
 ## 7. Test
 
-Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker locally before deploying to Cloudflare's global network.
+Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker locally before deploying to Cloudflare's global network. Take into account that there's no `local` mode support so `--remote` is required.
```
Comment (deadlypants1973) [inline content/browser-rendering/get-started/browser-rendering-with-DO.md]:
```suggestion
Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker remotely before deploying to Cloudflare's global network. Local mode support does not exist for Browser Rendering so `--remote` is required.
```

File: content/browser-rendering/get-started/reuse-sessions.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ Besides `puppeteer.sessions()`, we've added other methods to facilitate session
 
 ## 5. Test
 
-Run `npx wrangler dev --remote` to test your Worker locally before deploying to Cloudflare's global network.
+Run `npx wrangler dev --remote` to test your Worker locally before deploying to Cloudflare's global network. Take into account that there's no `local` mode support so `--remote` is required.
```
Comment (deadlypants1973) [inline content/browser-rendering/get-started/reuse-sessions.md]:
```suggestion
Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker remotely before deploying to Cloudflare's global network. Local mode support does not exist for Browser Rendering so `--remote` is required.
```

File: content/browser-rendering/get-started/screenshots.md [hunk 4]
```diff
@@ -173,7 +173,7 @@ If the same `"url"` is requested again, it will use the cached version in KV ins
 
 ## 6. Test
 
-Run `npx wrangler dev --remote` to test your Worker locally before deploying to Cloudflare's global network.
+Run `npx wrangler dev --remote` to test your Worker locally before deploying to Cloudflare's global network. Take into account that there's no `local` mode support so `--remote` is required.
```
Comment (deadlypants1973) [inline content/browser-rendering/get-started/screenshots.md]:
```suggestion
Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker remotely before deploying to Cloudflare's global network. Local mode support does not exist for Browser Rendering so `--remote` is required.
```

File: content/browser-rendering/platform/wrangler.md [hunk 5]
```diff
@@ -43,3 +43,5 @@ After the binding is declared, access the DevTools endpoint using `env.MYBROWSER
 ```javascript
 const browser = await puppeteer.launch(env.MYBROWSER);
 ```
+
+To test locally use `npx wrangler dev --remote` (there's no full local support yet, so `--remote` is required) or, to deploy, use `npx wrangler deploy`.
```
Comment (deadlypants1973) [inline content/browser-rendering/platform/wrangler.md]:
```suggestion
Run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker remotely before deploying to Cloudflare's global network. Local mode support does not exist for Browser Rendering so `--remote` is required. To deploy, run [`npx wrangler deploy`](/workers/wrangler/commands/#deploy).
```

**Review by celso — APPROVED** [review]

**Pull request merged**